app/views/job_invocations/_tab_hosts.html.erb

```diff
@@ -25,11 +25,12 @@
     </thead>
 
     <tbody>
+    <% parent_task = @job_invocation.task %>
```

Comment:
Do we need this variable?

Reply:
if we insist on the variable, we would need to add it to the
https://github.com/theforeman/foreman_remote_execution/blob/734acd451fac04ab0ebb9c348271bc66c69df3b8/app/views/job_invocations/show.js.erb#L14